crypto/Pos/Crypto/Signing/Types/Signing.hs

```diff
@@ -45,6 +46,7 @@ import           Pos.Crypto.Hashing (hash)
 -- Orphan instances
 ----------------------------------------------------------------------------
 
+#if !MIN_VERSION_cardano_crypto(1,1,0)
```

Comment:
What's the motivation for this? Are this instances included in newer cardano_crypto?